CommonControls/FileTypes/Animation/AnimationFile.cs

```diff
@@ -169,6 +169,17 @@ static AnimationHeader GetAnimationHeader(ByteChunk chunk)
             return header;
         }
 
+        public int GetIdFromBoneName(string name)
+        {
+            var result = Bones.Where(x => x.Name.Equals(name, StringComparison.OrdinalIgnoreCase));
```

Comment:
Use string.Compare instead of string.Equals. They behave the same, but most people would expect the former so it makes the code eaiser to read for others. You also want to use StringComparisong.InvariantCultureIgnoreCase